recipes/whisper-cpp/all/conanfile.py

```diff
@@ -114,13 +152,19 @@ def generate(self):
         if self.options.no_f16c:
             tc.variables["WHISPER_NO_F16C"] = True
 
+        if self.options.get_safe("with_openvino"):
+            tc.variables["WHISPER_OPENVINO"] = True
+            tc.variables["CMAKE_CXX_STANDARD"] = self.settings.get_safe("compiler.cppstd", 11)
```

Comment:
CMakeToolchain is already setting the `CMAKE_CXX_STANDARD` variable:
```suggestion
        tc.variables["WHISPER_OPENVINO"] = self.options.get_safe("with_openvino")
```

Reply:
> CMakeToolchain is already setting the `CMAKE_CXX_STANDARD` variable:

not for Conan 1.x

Reply:
@franramirez688 the problem is that here in CCI and the default behavior, the cppstd is not configured 😞